### ISSUE TITLE
Introduce SW False Positive Atlas, as a Product

### ISSUE THREAD

**Issue by drphilmarshall** [drphilmarshall/SpaceWarps#42]
I guess we'll host the catalog and associated PNGs (plus a nice PDF document of them all?) on a website somewhere. I wonder if the zooniverse would be happy to make us a (for now) unlisted folder at spacewarps.org/products where we can put things like this for people to download easily? Is this something to design and then ask for?

**Comment by drphilmarshall** [comment]
Re-starting this issue, after @cpadavis assembled the ML training set for the Swinburne group. I got a request from Andy Connolly at UW for a set of gravitational lens images, for ML classification investigation, and I expect others will be forthcoming. I think we should make a simple webpage, linked from our [github.io home page](http://drphilmarshall.github.io/SpaceWarps/), that introduces the image set that Chris made, and provides a link to the tarball for download. If this sounds OK to you, I'll make it and close this issue. I guess we might want to advertize it from the main app as well - and certainly the README of this repo. Thoughts?

**Comment by anupreeta27** [comment]
Ok. I don't know what exactly @cpadavis provides but the images+catalog of all SW identified known and new candidates are here - 
https://github.com/drphilmarshall/SpaceWarps/tree/master/analysis/projects/CFHTLS/candidates

And, I see that the above is not linked from the main app. Should we link this as well?

**Comment by aprajita** [comment]
seems reasonable to link both files and provide the info on a linked web page.

I might recall incorrectly, but is Chris' dataset based on user click clouds, so is dominated by FPs (with some real lenses) but not duds - or does he also include inspected red galaxies/groups without any features or  in his training set that would also help ML training sets. I thnk we had a conversation when I visited SLAC about using every galaxy in the images Chris was wusing to add these field galaxies as duds for  ML training set - was this done?

**Comment by drphilmarshall** [comment]
No! It was not - at the moment, we just have features clicked by
volunteers, including a mixture of expert-classified lenses and
expert-rejected false positives. I guess some of the latter will be LRGs
without lensing features but it would be good if we included more of these,
I agree. I'd suggest that we post the set that @cpadavis made (and sent to
Karl and Colin), and then we iterate later. Good call linking to the
candidates as well. I think we should follow Chris' slides and include the
example figures from the paper, too - I'll do this. Thanks!
